pephub/routers/api/v1/project.py

```diff
@@ -1138,3 +1139,33 @@ def delete_full_history(
             status_code=400,
             detail="Could not delete history. Server error.",
         )
+
+
+@project.get(
+    "/standardize",
+    summary="Standardize PEP metadata column headers",
+)
+async def get_standardized_cols(
+    namespace: str, project: str, tag: Optional[str] = DEFAULT_TAG, schema: str = ""
+):
+    """
+    Standardize PEP metadata column headers using BEDmess.
+
+    Args:
+    - pep (str): PEP string to be standardized
+    - schema (str): Schema for AttrStandardizer
```

Comment:
We are using `restructured docstring`, look on other methods

Reply:
added